client/src/components/Login.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { signApi } from "../api/auth";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
+  const handleChange = event => {
+    if (event.target.name === "email") {
+      setEmail(event.target.value);
+    }
+    if (event.target.name === "password") {
+      setPassword(event.target.value);
+    }
+  };
+  const signIn = () => {
+    signApi(email, password)
+      .then(data => {
+        const token = data.token;
+        console.log(token);
```

Comment:
It's generally good practise to remove all `console.log`s from the frontend, so let's remove it here